chain/chain/src/tests/.simple_chain.rs.pending-snap

```diff
@@ -0,0 +1,7 @@
+{"run_id":"030eb15a-05e2-4fe4-9dfc-047a5ff2ad70","line":38,"new":{"module_name":"near_chain__tests__simple_chain","snapshot_name":"build_chain","metadata":{"source":"chain/chain/src/tests/simple_chain.rs","assertion_line":38,"expression":"hash"},"snapshot":"A18WR4cyLHU3osVajUFknmfygvroERSquoKPAMXuFWTT"},"old":{"module_name":"near_chain__tests__simple_chain","metadata":{},"snapshot":"GHZFAFiMdGzAfnWTcS9u9wqFvxMrgFpyEr6Use7jk2Lo"}}
```

Comment:
Is this file added on purpose? I'm not super familiar with what these snap files are supposed to mean, so forgive my ignorance :D